apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -137,3 +137,35 @@ function wpcom_global_styles_override_for_free_site( $blog_id = 0 ) {
 	}
 }
 add_action( 'wp_print_styles', 'wpcom_global_styles_override_for_free_site' );
+
+/**
+ * Tracks when global styles are updated or reset after the post has actually been saved.
+ *
+ * @param int     $blog_id Blog ID.
+ * @param WP_Post $post    Post data.
+ * @param bool    $updated This value is true if the post existed and was updated.
+ */
+function wpcom_track_global_styles( $blog_id, $post, $updated ) {
+	require_lib( 'tracks/client' );
```

Comment:
I don't think this is available on Atomic sites and you need to use `wpcomsh_record_tracks_event` instead.

Reply:
Updated so that we can invoke the proper function based on the infrastructure.